04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,63 @@
+const Product = require('../models/product')
+
+
+const getAllProductsStatic = async (req,res) => {
+    const search = 'ab'
+    const products = await Product.find({price:{$gt: 30}}).sort('price').select('name price')
+    res.status(200).json({ products, ngHits: products.length })
+}
+const getAllProducts = async (req,res) => {
+    const { featured,company, name, sort, fields, numericFilters } = req.query
```

Comment:
challenge question: can you add a new query parameter? For example, minimumNameLength=8 could filter results whose name is shorter than 8 letter long.